### DISCUSSION TITLE
Mac electron build issue with default project

### DISCUSSION THREAD

**Discussion by ap1969** [quasarframework/quasar#9109] [CLI - Electron mode]
Hi,
I'm struggling to get an electron project built and running on the mac, so I started from scratch:

```
quasar create testproject
quasar mode add electron
quasar build -m electron 
```

That created the Unpackaged folder, but not the packaged one.

So in quasar.conf.js I switched the bundler to "builder".

again ran 
`quasar build -m electron `

Now I have the Packaged folder.

However, when I drag the dmg folder to Applications, and try to run it, it crashes unexpectedly.

The error is:
```
Process:               Mobi Folders [58272]
Path:                  /Applications/Mobi Folders.app/Contents/MacOS/Mobi Folders
Identifier:            com.mobifolders.app
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Mobi Folders [58272]
User ID:               501

Date/Time:             2021-04-29 18:05:24.255 -0500
OS Version:            macOS 11.0.1 (20B29)
Report Version:        12
Anonymous UUID:        D84140E9-FB8D-4450-A0E7-76B213A9686D


Time Awake Since Boot: 1400000 seconds

System Integrity Protection: disabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (Code Signature Invalid)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

**Termination Reason:    Namespace CODESIGNING, Code 0x1**

```

Any pointers on what to do next?

Regards,
Andy

**Comment by ontwikkelfabriek** [comment]
Here's an Apple FAQ question about it:
https://developer.apple.com/library/archive/qa/qa1884/_index.html

and an Electron github thread:
https://github.com/electron/electron-osx-sign/issues/130

**Reply by ap1969** [reply]
Ah, thanks so much. I'll take a look at the electron thread later today, see if their fix works.
